### PR TITLE
2018 add disease factor

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/Db/Diff/PKSimDB_diff.txt [hunk 1]
```diff
@@ -1,9 +1,13 @@
+UPDATE tab_calculation_method_rates SET formula='f_onto * f_disease * C_0 * V' WHERE calculation_method='CompoundCommon' AND formula_rate='PARAM_StartAmount_Protein';
```
Comment (msevestre) [inline src/Db/Diff/PKSimDB_diff.txt:1]: @Yuri05 I have added the f_disease parameter along side f_onto

File: src/PKSim.Core/Model/IndividualMolecule.cs [hunk 2]
```diff
@@ -24,26 +25,35 @@ public virtual double OntogenyFactor
...
          get => OntogenyFactorGIParameter.Value;
          set => OntogenyFactorGIParameter.Value = value;
       }
 
-      public IParameter OntogenyFactorGIParameter => this.Parameter(CoreConstants.Parameters.ONTOGENY_FACTOR_GI);
+      public IParameter OntogenyFactorGIParameter => this.Parameter(ONTOGENY_FACTOR_GI);
+
+      public IParameter DiseaseFactorParameter => this.Parameter(DISEASE_FACTOR);
+
+      public virtual double DiseaseFactor
```
Comment (msevestre) [inline src/PKSim.Core/Model/IndividualMolecule.cs:40]: This parameter can be accessed directly in the different implementation to update the value based on the disease state if required.

File: src/PKSim.Core/Services/CKDDiseaseStateImplementation.cs [hunk 3]
```diff
@@ -215,6 +215,11 @@ public void Validate(OriginData originData)
          return (false, PKSimConstants.Error.CKDOnlyAvailableForAdult);
       }
 
+      public void ApplyTo(IndividualMolecule individualMolecule)
+      {
```
Comment (msevestre) [inline src/PKSim.Core/Services/CKDDiseaseStateImplementation.cs:219]: CKD does not do anything FOR NOW. There is a publication coming out soon influencing some enzymes and transporters concentrations

File: src/PKSim.Core/Services/ExpressionProfileUpdater.cs [hunk 4]
```diff
@@ -150,6 +153,15 @@ public void SynchroniseSimulationSubjectWithExpressionProfile(ISimulationSubject
          var (sourceMolecule, sourceIndividual) = expressionProfile;
          // ExpressionProfile => SimulationSubject, we want to make sure that the parameters in simulation subject are linked to their expression profile origin parameters
          synchronizeExpressionProfiles(sourceMolecule, sourceIndividual, moleculeInIndividual, simulationSubject, updateParameterOriginId: true);
+
+         //Once the synchronization was performed, apply changes to simulation subject molecules based on disease state
+         updateMoleculeParametersForDiseaseState(simulationSubject, moleculeInIndividual);
+      }
+
+      private void updateMoleculeParametersForDiseaseState(ISimulationSubject simulationSubject, IndividualMolecule moleculeInIndividual)
+      {
```
Comment (msevestre) [inline src/PKSim.Core/Services/ExpressionProfileUpdater.cs:162]: After everything has been updated, we apply the change again. This is a fairly straight forward implementaiton. We might want to do something more "clever" later on if needed.

File: src/PKSim.Core/Services/IndividualMoleculeFactory.cs [hunk 5]
```diff
@@ -125,14 +125,15 @@ protected TMolecule CreateMolecule(string moleculeName)
          CreateMoleculeParameterIn(molecule, REFERENCE_CONCENTRATION, CoreConstants.DEFAULT_REFERENCE_CONCENTRATION_VALUE, MOLAR_CONCENTRATION);
```
Comment (msevestre) [inline src/PKSim.Core/Services/IndividualMoleculeFactory.cs:125]: We need to read this from the DB. We are duplicating so much info it does not make sense

File: src/PKSim/Program.cs [hunk 6]
```diff
@@ -20,7 +20,7 @@ private static void Main(string[] args)
          Application.EnableVisualStyles();
          Application.SetCompatibleTextRenderingDefault(false);
 
-         //   WindowsFormsSettings.SetDPIAware();
+         WindowsFormsSettings.SetDPIAware();
```
Comment (msevestre) [inline src/PKSim/Program.cs:23]: on my laptop, PKSim is not usable without this setting turned on (everyhting is blurry).

**Review by msevestre — COMMENTED** [review]
Fixes #2018

**Comment by msevestre** [comment]
@Yuri05 I am going to merge this for now and trigger a build on the ckd feature branch

File: src/Db/Diff/PKSimDB_diff.txt [hunk 7]
```diff
@@ -1,9 +1,13 @@
+UPDATE tab_calculation_method_rates SET formula='f_onto * f_disease * C_0 * V' WHERE calculation_method='CompoundCommon' AND formula_rate='PARAM_StartAmount_Protein';
+INSERT INTO tab_calculation_method_rates(calculation_method,formula_rate,formula,dimension) VALUES('DiseaseStates','One_Rate','1','Fraction');
```
Comment (Yuri05) [inline src/Db/Diff/PKSimDB_diff.txt:2]: I think the dimension of the "disease factor" and its formula should be `dimensionless`, not `fraction`.
Especially the factor could become also >1

Comment (Yuri05) [reply src/Db/Diff/PKSimDB_diff.txt:2]: yep, Ontogeny factor should be dimensionless as well...

**Pull request merged**